src/bundle/Resources/views/themes/admin/ui/component/side_panel/side_panel.html.twig

```diff
@@ -0,0 +1,47 @@
+{% import '@ibexadesign/ui/component/macros.html.twig' as html %}
+
+{% trans_default_domain 'ibexa_admin_ui' %}
+
+{% set config_panel_main_class = 'ibexa-side-panel ibexa-side-panel--hidden' %}
+{% set config_panel_footer_class = 'ibexa-side-panel__footer' ~ (footer_class is defined ? footer_class ~ '')%}
```

Comment:
I would prefer to have the whole `attr_footer` so that in the future it would be easier to deal with.